bower.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "tota11y",
```

Comment:
Indeed. We should add it, yes. For some reason, bower init didn't ask me for one, weird...